lib/master.js

```diff
@@ -207,8 +223,9 @@ class Master extends BaseService {
                     resolve(this._startWorker(workerId));
                 });
             };
+            worker.on('exit', this._startupWorkerExitHandler);
 
-            const workerExit = (worker) => {
+            this._workerExitHandler = (worker) => {
```

Comment:
It was correct how it was.

Reply:
workerExit is removed in _stopWorker. So the local var would be out scope.

Reply:
And as we have discussed it's incorrect to remove it there.

Reply:
And as I have explained, we don't need to do it there. The workerExit must be removed from within a workerExit